### PR TITLE
Update microsoft-365-groups-expiration-policy.md

### PR DESCRIPTION
**Pull request by josephine-jijitech**
Updated the recent Yammer activities that triggers auto-renewal

### CODE REVIEW

File: microsoft-365/solutions/microsoft-365-groups-expiration-policy.md [hunk 1]
```diff
@@ -33,8 +33,7 @@ Groups that are actively in use are renewed automatically. Any of the following
 - SharePoint - view, edit, download, move, share, or upload files. (Viewing a SharePoint page does not count as an action for automatic renewal.)
 - Outlook - join group, read or write group message from the group, and like a message (Outlook on the web).
 - Teams - visiting a teams channel.
-
-Note that the only Yammer activity that will trigger an automatic group renewal is the upload of a document to SharePoint within the community.
+- Yammer - upload of a document to SharePoint within the community or view of a post within a Yammer community or an interactive email in Outlook.
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/microsoft-365-groups-expiration-policy.md:36]:
```suggestion
- Yammer - upload a document to SharePoint within the community or view a post within a Yammer community or an interactive email in Outlook.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @jose-smg, kindly commit the minor suggested change. Many thanks.

**Comment by yogkumgit** [comment]
@jose-smg Gentle reminder, please implement the suggested changes. Thanks

**Comment by yogkumgit** [comment]
@josephine-jijitech closing the PR, as suggested changes are not implemented. Please feel free to reopen it, in case you wish to take an action. Thanks

**Pull request closed without merging**